README.md

```diff
@@ -251,11 +251,11 @@ If you want to force a service to restart even if its service configuration hasn
 
 ### Service dependencies
 
-Pebble takes service dependencies into account when starting and stopping services. When Pebble starts a service, it also starts the services which that service depends on (configured with `required`). Conversely, when stopping a service, Pebble also stops services which depend on that service.
+Pebble takes service dependencies into account when starting and stopping services. When Pebble starts a service, it also starts the services which that service depends on (configured with `requires`). Conversely, when stopping a service, Pebble also stops services which depend on that service.
```

Comment:
Possibly off-topic:

I wonder if this would read a little better with a small doze or general computing jargon, for example:

"services which that service depends on" -> "dependencies"

Additionally, I'm left to guess what "also starts x" means, does it means that dependencies are started before or in parallel with the denepdent? Does pebble wait for dependencies to actually start?

Reply:
1) I intended to *define* the term "service dependencies" that I lead with, hence me spelling it out "which that service depends on". So I think I'll keep that.
2) Regarding "also starts", I do think that is clarified in the next paragraphs. `requires` doesn't actually do anything about ordering, just "starts with" or "also starts". `before` and `after` do the ordering part (kind of -- see the "of limited usefulness" note).